apps/system/index.html

```diff
@@ -280,9 +282,11 @@
                 <!-- App install manager notification -->
                 <div id="install-manager-notification-container">
                 </div>
+<!--IFNDEF_NO_BLUETOOTH
                 <!-- Bluetooth transfer -->
```

Comment:
ouch it makes editor's highlight looks bad...maybe we should keep it out of the preprocesser...or find out if there's a better alternative.